icekit/project/settings/_base.py

```diff
@@ -414,18 +414,18 @@
 # given model/field, without generating additional unecessary thumbnails. This
 # is essential when using a remote storage backend.
 THUMBNAIL_ALIASES = {
-#     'app[.model][.field]': {
-#         'name-WxH': {
-#             'size': (W, H),
-#             ...,
-#         },
-#     },
+    # 'app[.model][.field]': {
+    #   'name-WxH': { 'size': (W, H), },
+    # },
     '': {
+        'author_portrait': {
+            'size': (360, 640),
+        },
```

Comment:
When we receive a design, the author_portrait will normally always be the same size throughout designs, and I bet normally not 360x640. If it needs to be a different size than 360x640 for a given project then we only override the alias in the project settings and the change would be reflected in all templates and apps for that project.

If we put WxH in the alias name, then that name would become obsolete in every project, and we'd need to define a new alias and update/override all templates to use a new alias name.